main.py

```diff
@@ -0,0 +1,37 @@
+#importing the necessary libraries
+import csv
+
+#creating the variables to store the individual values
+temperature = []
+humidity = []
+
+#to open the csv file
+with open("weather_data.csv", "r") as csvfile:
+    reader = csv.reader(csvfile)
+    next(reader)  #to skip the header row
+
+    for row in reader:
+        temperature.append(float(row[1]))
+        humidity.append(int(row[2]))
+
+# #to sort the temperature and calculate the max and min
+# sorted_temperature = sorted(temperature)
+# max_temperature = sorted_temperature[-1]
+# min_temperature = sorted_temperature[0]
+# print(max_temperature)
+# print(min_temperature)
```

Comment:
These stray comments can go too! I know it seems knit-picky, but best to be aware of it for when you move to bigger codebases!